### ISSUE TITLE
virtualenv + Python 3 doesn't work

### ISSUE THREAD

**Issue by kmike** [rtfd/readthedocs.org#2504]
## Details

* Project URL:  https://readthedocs.org/projects/eli5
* Build URL (if applicable): https://readthedocs.org/projects/eli5/builds/4664724/
* Read the Docs username (if applicable): kmike

## Expected Result

I want to build docs, installing the package in a virtualenv using Python 3.

## Actual Result

Build fails:

```
Using base prefix '/usr'
New python executable in /home/docs/checkouts/readthedocs.org/user_builds/eli5/envs/latest/bin/python3
Not overwriting existing python script /home/docs/checkouts/readthedocs.org/user_builds/eli5/envs/latest/bin/python (you must use /home/docs/checkouts/readthedocs.org/user_builds/eli5/envs/latest/bin/python3)
Installing setuptools, pip, wheel...
  Complete output from command /home/docs/checkouts...s/latest/bin/python3 - setuptools pip wheel:
  Traceback (most recent call last):
  File "<stdin>", line 9, in <module>
  File "/usr/lib/python3.4/pkgutil.py", line 611, in get_data
    spec = importlib.util.find_spec(package)
  File "/home/docs/checkouts/readthedocs.org/user_builds/eli5/envs/latest/lib/python3.4/importlib/util.py", line 86, in find_spec
    parent = __import__(parent_name, fromlist=['__path__'])
ImportError: No module named 'pip._vendor'
```

**Comment by agjohnson** [comment]
If this was a project that was first set up as a python 2 project, perhaps there are some issues with the virtualenv. You can attempt to wipe the version from your version administration page.

Past that, not sure this is a problem with anything we're doing. Any background into why pip is triggering this would be helpful here.

**Comment by kmike** [comment]
It doesn't seem to be related to virtualenv after all, it fails with the same error if I uncheck "install package to virtualenv" checkbox and use Python 3 as interpreter.

**Comment by kmike** [comment]
One of the early builds was with "Give the virtual environment access to the global site-packages dir." checkbox turned on. Other than that I only checked an unchecked "Install your project inside a virtualenv using setup.py install" checkbox and switched interpreters several times.

**Comment by agjohnson** [comment]
Regardless of whether you have use virtualenv checked, we install a virtualenv. I believe this is still related to the fact that this was built previously using python2.7. Try to wipe this version in the version admin page.

**Comment by humitos** [comment]
It seems the problem is fixed. Last builds are passing without any problem. Closing.

Feel free to reopen if you consider. Thanks!
